internal/distro/rhel9/imagetype.go

```diff
@@ -381,6 +382,12 @@ func (t *imageType) checkOptions(customizations *blueprint.Customizations, optio
 		return fmt.Errorf("edge raw images require specifying a URL from which to retrieve the OSTree commit")
 	}
 
+	// warn that user customizations on edge-commit, edge-container are deprecated
+	// TODO(edge): directly error if these options are provided when rhel-9.5's time arrives
+	if (t.name == "edge-commit" || t.name == "edge-container") && customizations.GetUsers() != nil {
+		log.Printf("Please note that user customizations on %q image type are deprecated and will be removed in the near future\n", t.name)
```

Comment:
do we need this? isn't this coming from weldr or something?

Reply:
yes, I'm working on it, I've left it as it is for the time being

Reply:
but I mean, the thing is that the warning will originate from osbuild-composer, which has to be sent back to weldr